### PR TITLE
fix keyboard shortcut for queue button

### PR DESCRIPTION
**Pull request by ajiou**
fix issue #1406

### CODE REVIEW

File: Extensions/keyboardShortcut.js [hunk 1]
```diff
@@ -109,7 +109,7 @@
     }
 
     function clickQueueButton() {
-        document.querySelector(".control-button-wrapper .spoticon-queue-16").click();
+        document.querySelector(".control-button-wrapper > button.Button-sc-1dqy6lx-0").click();
```
Comment (afonsojramos) [inline Extensions/keyboardShortcut.js:112]: It seems that the class being used is not mapped, so let's try to use a generic way of accessing this button, ie, simply this:
```js
document.querySelector(".control-button-wrapper > button").click();
```

**Review by afonsojramos — CHANGES_REQUESTED** [review]
@ajiou I've never used keyboard shortcuts, can you tell me how to test this?

File: Extensions/keyboardShortcut.js (hunk 1 above)
Comment (ajiou) [reply Extensions/keyboardShortcut.js]: I see. Also, I discovered that this selector would select the "Remove" button first, so i make it a bit more specific.

Comment (afonsojramos) [reply Extensions/keyboardShortcut.js]: The remove button? Which remove button? If that's the case then maybe we do need to be more specific.

Comment (ajiou) [reply Extensions/keyboardShortcut.js]: The one on the `.main-nowPlayingBar-left` to make the auto-suggested song not to play again
image

File: Extensions/keyboardShortcut.js [hunk 2]
```diff
@@ -109,7 +109,7 @@
     }
 
     function clickQueueButton() {
-        document.querySelector(".control-button-wrapper .spoticon-queue-16").click();
+        document.querySelector(".main-nowPlayingBar-right > div.ExtraControls > div.GlueDropTarget > div.control-button-wrapper > button").click();
```
Comment (afonsojramos) [inline Extensions/keyboardShortcut.js:112]: Let's keep it simple to reduce our dependency on the CSS Mappings: `div.ExtraControls div.control-button-wrapper > button`

**Review by afonsojramos — CHANGES_REQUESTED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**